src/index.js

```diff
@@ -1,23 +1,26 @@
 import { bindEvent, unbindEvent } from './main'
 
-const buildDirective = function (alias = {}) {
+const DEFAULT_FORBIDDEN_NODES = ['INPUT', 'TEXTAREA', 'SELECT']
+
+const buildDirective = function (alias = {}, forbiddenNodes = DEFAULT_FORBIDDEN_NODES) {
```

Comment:
Don't need the default here, if you set it on [here](https://github.com/Dafrok/v-hotkey/pull/42/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R22)
```suggestion
const buildDirective = function (alias = {}, forbiddenNodes) {
```